docs/Concepts/DevOps.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+title: DevOps
+nav_order: 1
+parent: Concepts
+has_children: false
+---
+
+# {{ page.title }}
```

Comment:
There shouldn't be an Hastag  " # " before the call  Use 
` {{ page.title }}
====================== `
 instead unless the title differs from the Title of the page within the URL and nagivation.

Reply:
Just pushed the fix, but this has already been merged.